docs/setup/kubeedge_install.md

```diff
@@ -0,0 +1,74 @@
+# Install KubeEdge
+
+In this section, we would cover the below topics
+
+1. [Abstract](#Abstract)
+2. [Pre-requisties](#prerequisties)
+3. [Setup KubeEdge](#setup-kubeEdge)
+4. [Configure KubeEdge](#configure-kubeEdge)
+5. [Run KubeEdge](#run-kubeedge)
+6. [KubeEdge Pre-Check](#kubeedge-pre-check)
+## Abstract
+KubeEdge is composed of cloud and edge parts. It is built upon Kubernetes and provides core infrastructure support for networking, application deployment and metadata synchronization between cloud and edge. So if we want to setup KubeEdge, we need to setup Kubernetes cluster, cloud side and edge side.
```

Comment:
**cloud part** or **Cloud part**? OR **cloud side**  or **Cloud side**
**edge part** or **Edge part** ? OR **edge side** or **Edge side**
Please confirm this and make sure it is consistently used across the docs.